digiroad2-api-oth/src/main/scala/fi/liikennevirasto/digiroad2/LaneApi.scala

```diff
@@ -9,12 +9,14 @@ import fi.liikennevirasto.digiroad2.service.lane.LaneService
 import fi.liikennevirasto.digiroad2.service.{RoadAddressService, RoadLinkService}
 import fi.liikennevirasto.digiroad2.util.LaneUtils.pwLanesTwoDigitLaneCode
 import fi.liikennevirasto.digiroad2.util.RoadAddress.{isCarTrafficRoadAddress, roadPartNumberRange}
-import fi.liikennevirasto.digiroad2.util.{PolygonTools, RoadAddressRange, Track}
+import fi.liikennevirasto.digiroad2.util.{PolygonTools, RoadAddressException, RoadAddressRange, Track}
 import org.json4s.{DefaultFormats, Formats}
 import org.scalatra.json.JacksonJsonSupport
 import org.scalatra.swagger.{Swagger, SwaggerSupport}
 import org.scalatra.{BadRequest, ScalatraServlet}
 
+import scala.collection.mutable
```

Comment:
Ei nyt kovin kriittistä, mut varmaan tästä vois importata pelkän linkedhashmapin, ja käyttää suoraan sitä koodissa.

Reply:
Tein noin ja IntelliJ värjäsi nämä kohdat silloin keltaisella ja neuvoi käyttämään mutable prefix, itsellä käytössä default asetukset noissa IntelliJ vihjeissä, en sitten tiedä onko oikeasti parempi tehdä näin, mutta ainakaan intelliJ ei herjaa.